acceptance/setup/packages/010_Install.rb

```diff
@@ -32,15 +41,20 @@
 # Install puppet on all hosts including master
 install_packages_on(hosts, PACKAGES)
 
+step 'Install build dependencies on master'
+
+MASTER_PACKAGES = {
+  :redhat => [
+    'git',
+    'java-1.8.0-openjdk-devel',
+  ],
+}
+install_packages_on(master, MASTER_PACKAGES, :check_if_exists => true)
+
 step 'Install MSIs on any Windows agents'
 agents.each do |agent|
   if agent.platform.start_with?('windows')
     logger.info "Installing Puppet agent msi #{sha} on #{agent}"
     install_puppet_agent_dev_repo_on(agent, :version => sha)
-    logger.info 'Prevent Puppet Service from Running'
```

Comment:
This is unnecessary, as Beaker automatically sets a startup type of `Manual` by default - see https://github.com/puppetlabs/beaker/blob/master/lib/beaker/dsl/install_utils/windows_utils.rb#L111

If you look at the Beaker logs, you will also see something like this:

```
localhost $ scp /var/folders/1b/rzv96bhd5h9bwtdnz7g0xmlm0000gn/T/install-puppet-msi-2015-10-23_18.17.28.bat20151023-36854-1rusm7n win2012r2-64-1:C:\Windows\TEMP\install-puppet-msi-2015-10-23_18.17.28.bat {:ignore => }

pj6odfexaonohan.delivery.puppetlabs.net (win2012r2-64-1) 18:17:28$ cmd.exe /c "C:\Windows\TEMP\install-puppet-msi-2015-10-23_18.17.28.bat"

  C:\cygwin64\home\Administrator>start /w msiexec.exe /i "C:\PROGRA~3\puppet-agent-x64.msi" /qn /L*V C:\Windows\TEMP\install-puppet-2015-10-23_18.17.28.log PUPPET_AGENT_STARTUP_MODE=Manual

  C:\cygwin64\home\Administrator>exit /B 0

pj6odfexaonohan.delivery.puppetlabs.net (win2012r2-64-1) executed in 28.92 seconds

pj6odfexaonohan.delivery.puppetlabs.net (win2012r2-64-1) 18:17:57$ cmd.exe /c sc query puppet || sc query pe-puppet

  SERVICE_NAME: puppet
          TYPE               : 10  WIN32_OWN_PROCESS
          STATE              : 1  STOPPED
          WIN32_EXIT_CODE    : 1077  (0x435)
          SERVICE_EXIT_CODE  : 0  (0x0)
          CHECKPOINT         : 0x0
          WAIT_HINT          : 0x0

pj6odfexaonohan.delivery.puppetlabs.net (win2012r2-64-1) executed in 0.11 seconds
```